rtengine/CMakeLists.txt

```diff
@@ -20,12 +20,17 @@ set (RTENGINESOURCEFILES colortemp.cc curves.cc flatcurves.cc diagonalcurves.cc
     dirpyr_equalizer.cc
     calc_distort.cc lcp.cc dcp.cc ipretinex.cc
     cJSON.c camconst.cc
-    klt/convolve.cc klt/error.cc klt/klt.cc klt/klt_util.cc klt/pnmio.cc klt/pyramid.cc klt/selectGoodFeatures.cc
-    klt/storeFeatures.cc klt/trackFeatures.cc klt/writeFeatures.cc
     clutstore.cc
     ciecam02.cc
     )
 
+IF (NOT KLT_FOUND)
```

Comment:
@sharkcz Keywords/function calls in this file are lower case, so it should read `if (...)` instead of `IF (...)`. And while you're at it, you could also get rid of the pre-CMake-2.6 repetition of the condition in `else (...)` and `endif (...)`.

Best
Flössie